crates/protogen/src/metastore/types/catalog.rs

```diff
@@ -494,47 +494,38 @@ impl From<FunctionType> for catalog::function_entry::FunctionType {
         }
     }
 }
+
```

Comment:
i was getting some inconsistencies when converting this at runtime, so i modified it to be a bit more explicitly linked to the proto definition.